src/bookmarksList.html

```diff
@@ -9,13 +9,13 @@
 
   <body ng-controller="bookmarksCtrl">
 
-    <div class="bar bar-footer bar-calm item-input-inset">
+    <div class="bar bar-header bar-calm item-input-inset">
```

Comment:
No idea why it was `bar-footer` before since I always remember the bar being a header. Maybe something to do with me upgrading Ionic but it started actually appearing as a footer, and changing it to `bar-header` fixed that (amazing, isn't it?).